src/loader.py

```
@@ -196,30 +283,33 @@ def _clip_netcdf_cdo(path: Path, params: Params):
     selregion_cmd = f"-selregion,{params.base_path}/reference_area.ascii"
 
     # build the full command
-    cmd = ['cdo', selregion_cmd, lonlat_cmd, str(path), str(out_name)]
-    
+    cmd = ["cdo", selregion_cmd, lonlat_cmd, str(path), str(out_name)]
+
     # run
     t1 = time.time()
     subprocess.run(cmd)
```

Ruff (0.14.5)

`subprocess` call: check for execution of untrusted input

(S603)